go/lint-project.sh

```diff
@@ -51,6 +51,17 @@ then
     fi
 fi
 
+# Verify we're using the latest version of github.com/moovfinancial/events if it's a dependency
+eventsLibrary="github.com/moovfinancial/events"
+eventsVersion=$( go list -u -m -mod=mod $eventsLibrary | awk -F'[][]' '{print $2}')
+if [[ $eventsVersion ]]
+then
+    echo "$eventsLibrary needs to be updated to the latest release: $eventsVersion" 
```

Comment:
```suggestion
    echo "$eventsLibrary needs to be updated to the latest release: $eventsVersion" 
    echo "Run 'go get -u ""$eventsLibrary""@latest' to resolve this issue"
```

I think this is the right syntax to properly generate a command they can run.

Reply:
running this outputs
```bash
github.com/moovfinancial/events needs to be updated to the latest release: v0.4.12
Run 'go get -u github.com/moovfinancial/events@latest' to resolve this issue
```

Reply:
I'll update this to try to get it to send a link